src/atoms/ResponsibilityBox.jsx

```diff
@@ -57,5 +57,5 @@ export default function ResponsibilityBox({ responsibility }) {
 }
 
 ResponsibilityBox.propTypes = {
-  responsibility: PropTypes.oneOf(["none", "some", "all"]).isRequired,
+  responsibility: PropTypes.oneOf(["Inherited", "Hybrid", "Allocated"]),
```

Comment:
This should still have `.isRequired` tacked on to the end because this component will not be able to render without one of these props being passed in.

Reply:
Good callout, I removed it because the initial values were wrong and I forgot to add it back when I found all the correct values :)